app/channel.go

```diff
@@ -726,7 +726,7 @@ func (a *App) GetChannelModerationsForChannel(channel *model.Channel) ([]*model.
 	}
 
 	var higherScopedGuestRole *model.Role
-	if len(guestRoleName) > 0 {
+	if len(higherScopedGuestRoleName) > 0 {
```

Comment:
oh good catch - that would have been bad

Reply:
Yup, I added that bug on my last PR.